spyder/plugins/editor/widgets/tests/conftest.py

```diff
@@ -126,6 +126,37 @@ def teardown():
     return completions
 
 
+@pytest.fixture
+def mock_codeeditor(qtbot_module, request):
```

Comment:
Let's rename this to `mock_completions_codeeditor` because `mock_codeeditor` is too broad.